README.md

```diff
@@ -1,5 +1,14 @@
 # rhof
 
+[![Build](https://github.com/pwnb0y/rhof/actions/workflows/nodejs.yml/badge.svg)](https://github.com/pwnb0y/rhof/actions/workflows/nodejs.yml)
```

Comment:
`pwnb0y` => `gh-conf`

seems there is some issue with the `npm-badger` tool. It used your username.

Can you update this? Find and replace in the whole file

Reply:
@pwnb0y I think my comments were not clear.

I meant in these lines https://github.com/gh-conf/rhof/pull/50/commits/916dc6db699a2321cc12141443e5aad7d223208f#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3-R9

replace `pwnb0y` => `gh-conf`